resources/lang/id/user_verification.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'box' => [
+        'sent' => 'Kode verifikasi telah dikirim ke :mail. Masukkan kode yang diterima.',
+        'title' => 'Verifikasi Akun',
+        'verifying' => 'Memverifikasi...',
+        'issuing' => 'Meminta kode baru...',
+
+        'info' => [
+            'check_spam' => 'Pastikan untuk memeriksa folder spam Anda jika Anda tidak dapat menemukan emailnya.',
+            'recover' => 'Jika Anda tidak dapat mengakses email Anda atau lupa apa yang Anda gunakan, silahkan ikuti :link.',
+            'recover_link' => 'proses pemulihan email di sini',
+            'reissue' => 'Anda juga dapat :reissue_link atau :logout_link.',
+            'reissue_link' => 'meminta kode baru',
+            'logout_link' => 'keluar',
+        ],
+    ],
+
+    'email' => [
+        'subject' => 'verifikasi akun osu!',
+    ],
+
+    'errors' => [
+        'expired' => 'Kode verifikasi telah kadaluwarsa, email verifikasi baru telah dikirim.',
```

Comment:
Should be "kedaluwarsa" instead of "kadaluwarsa" here as Pyrozen stated earlier

Reply:
Belum diganti?

Reply:
Oh shoot, i forgot this one (i only applied it to store.php)